dali/python/nvidia/dali/plugin/tf.py

```diff
@@ -51,6 +51,14 @@
     differ based on your use case.
 """
 
+def serialize_pipeline(pipeline):
+  try:
+    return pipeline.serialize()
+  except:
+    print("Error during pipeline initialization. Note that some operators (e.g. Python Operators) "
+          "cannot be used with tensorflow data set API and DALIIterator.")
```

Comment:
Here I would propose modified exception message rather then print. It should play nicer with various debuggers and stuff, I think.

Reply:
I think we rethrow the caught exception?
You can try to catch the error and modify the message, but it may be about something else.

Reply:
changed to: 
```python
except RuntimeError as e:
    raise RuntimeError("Error during pipeline initialization. Note that some operators "
                       "(e.g. Python Operators) cannot be used with "
                       "tensorflow data set API and DALIIterator.") from e
```
the error message looks like that:
```
---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
<ipython-input-4-65ec889c17c7> in <module>
     20 try:
---> 21     pipe.serialize()
     22 except RuntimeError as e:

~/mydali/build/venv/lib/python3.6/site-packages/nvidia/dali/pipeline.py in serialize(self)
    540             self._pipe.SetOutputNames(self._names_and_devices)
--> 541         return self._pipe.SerializeToProtobuf()
    542 

RuntimeError: [/home/rafal/mydali/dali/pipeline/pipeline.cc:698] Assert on "spec.GetSchema().IsSerializable()" failed: Could not serialize the operator: DLTensorPythonFunctionImpl
Stacktrace (100 entries):
(the stacktrace I would like to get rid of) 

The above exception was the direct cause of the following exception:

RuntimeError                              Traceback (most recent call last)
<ipython-input-4-65ec889c17c7> in <module>
     23     raise RuntimeError("Error during pipeline initialization. Note that some operators "
     24                        "(e.g. Python Operators) cannot be used with "
---> 25                        "tensorflow data set API and DALIIterator.") from e    
     26 pipe.build()
     27 ims1, ims2 = pipe.run()

RuntimeError: Error during pipeline initialization. Note that some operators (e.g. Python Operators) cannot be used with tensorflow data set API and DALIIterator.
```

Reply:
Capitalization & maybe make it more clear about serialization & TF?
```
raise RuntimeError("Error during pipeline initialization. Note that some operators "
                       "that are not serializable (e.g. Python Operators) , cannot be used with "
                       "DALI TensorFlow Plugin for Dataset API and DALIIterator.")
```